extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -714,6 +714,12 @@ export class AmpStoryPage extends AMP.BaseElement {
       return new Promise((resolve) => {
         switch (mediaEl.tagName.toLowerCase()) {
           case 'amp-audio':
+            if (mediaEl.getAttribute('layout') === Layout.NODISPLAY) {
+              whenUpgradedToCustomElement(mediaEl)
+                .then((el) => el.signals().whenSignal(CommonSignals.BUILT))
+                .then(resolve, resolve);
+              break;
+            }
```

Comment:
It looks like amp-video supports `NODISPLAY` too... Could you move this block so we support both audio and video?

Reply:
I think doing something like `const signal = mediaEl.getAttribute('layout') === Layout.NODISPLAY ? BUILT : LOAD_END;` so you don't have to repeat this block might be easier, but it LGTM :))